packages/graphql/mixin.server.js

```diff
@@ -21,12 +21,19 @@ class GraphQLMixin extends Mixin {
     super(config, element);
 
     this.client = this.createClient(options);
-    this.shouldPrefetchOnServer = undefined;
+    this.prefetchOnServer =
```

Comment:
This also needs to happen in the `hops-redux` package.

Reply:
By default `prefetchOnServer` is `true` and we only do this, so that people can disable it entirely through their config.